website/docs/multiplatform-patching.md

```diff
@@ -50,6 +50,10 @@ If you don't provide a `--report` flag pointing to a directory, Copa will not pe
 
 If `--push` is not specified, the individual patched images will be saved locally, and you can push them to your registry later using `docker push` and then `docker manifest create/push` to create the multi-platform manifest.
 
+**Note:** Copa copies over unpatched platforms as a passthrough - only platforms with vulnerability reports are patched, while other platforms remain unchanged in the final multi-platform image.
+
+**⚠️ Warning:** Build attestations, signatures, and OCI referrers from the original image are not preserved or copied to the patched image.
```

Comment:
we can use docusaurus's built in syntax so it visualizes nicely https://docusaurus.io/docs/next/markdown-features/admonitions

```
:::warning
```

Reply:
fixed